src/NpmUtilities.js

```diff
@@ -76,21 +76,21 @@ export default class NpmUtilities {
   }
 
   @logger.logifySync()
-  static addDistTag(packageName, version, tag, registry) {
+  static addDistTag(directory, packageName, version, tag, registry) {
```

Comment:
How about specifying the [`cwd` option](https://nodejs.org/dist/latest-v6.x/docs/api/child_process.html#child_process_child_process_execsync_command_options) for `execSync` rather than the manual `cd`?